### PR TITLE
Forward configuration parameters.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: lib/rmt.toit [hunk 1]
```diff
@@ -258,23 +258,51 @@ class Channel:
...
 
   /**
   Variant of $(constructor pin).
 
-  Configures the channel for output. See $(configure --output).
+  Configures the channel for output. See $(configure --output) for output parameters.
   */
-  constructor --output/bool pin/gpio.Pin --memory_block_count/int=1 --channel_id/int?=null:
+  constructor --output/bool pin/gpio.Pin --memory_block_count/int=1
+      --channel_id /int? = null
```
Comment (kasperl) [inline lib/rmt.toit:288]: Let's make sure the type syntax (whitespace) is consistent here.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: lib/rmt.toit [hunk 2]
```diff
@@ -258,23 +258,51 @@ class Channel:
...
+      --clk_div /int = DEFAULT_OUT_CLK_DIV
+      --flags /int = DEFAULT_OUT_FLAGS
+      --enable_carrier /bool = DEFAULT_OUT_ENABLE_CARRIER
+      --carrier_frequency_hz /int = DEFAULT_OUT_CARRIER_FREQUENCY
+      --carrier_level /int = DEFAULT_OUT_CARRIER_LEVEL
+      --carrier_duty_percent /int = DEFAULT_OUT_CARRIER_DUTY_PERCENT
+      --idle_level/int? = DEFAULT_OUT_IDLE_LEVEL:
```
Comment (kasperl) [inline lib/rmt.toit:295]: Here!

Comment (floitsch) [reply lib/rmt.toit]: done.

**Pull request merged**